Python/PRP/GUI/pyGUIKnobCtrl.cpp

```diff
@@ -0,0 +1,115 @@
+/* This file is part of HSPlasma.
+*
+* HSPlasma is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* HSPlasma is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <PRP/GUI/pfGUIKnobCtrl.h>
+#include "pyGUIKnobCtrl.h"
+#include "pyGUIValueCtrl.h"
+#include "PRP/KeyedObject/pyKey.h"
+#include "Math/pyGeometry3.h"
+
+extern "C" {
+
+PY_PLASMA_NEW(GUIKnobCtrl, pfGUIKnobCtrl)
+
+PY_METHOD_VA(GUIKnobCtrl, addAnim,
```

Comment:
Sadly, the library is inconsistent in this naming, which threw me off.  We use this pattern so much, too.